src/platform/switch/switch_ui.cpp

```diff
@@ -241,6 +295,15 @@ NxUi::NxUi(int width, int height, const Game_ConfigVideo& cfg) : BaseUi(cfg)
 #endif
 	SetIsFullscreen(true);
 
+	auto ver = hosversionGet();
+	Output::Debug("Running on Nintendo Switch ({}.{}.{}{})",
+		HOSVER_MAJOR(ver), HOSVER_MINOR(ver), HOSVER_MICRO(ver),
+		(hosversionIsAtmosphere() ? " with Atmosphere" : ""));
+
+	appletLockExit();
+	appletHook(&applet_hook_cookie, appletHookCallback, nullptr);
+	appletSetFocusHandlingMode(AppletFocusHandlingMode_SuspendHomeSleepNotify);
```

Comment:
Retroarch does use `NoSuspend` mode and does own handling when in background. This means unfortunately more code in our backend, however might fix the audio issue after standby (because the player keeps running in idle, means more power consumption :/)